dietpi/patch_file

```diff
@@ -2650,6 +2650,12 @@ _EOF_
 		fi
 		#-------------------------------------------------------------------------------
 
+        elif (( $VERSION_CURRENT == 154 )); then
```

Comment:
This wont have any effect until v155 is released. As the current version is v153 during the v154 update.

@MichaIng 

If you could remove this section and place the patch code in ```elif (( $VERSION_CURRENT == 153 )); then```, i'd be grateful and merge this commit.